Project/Source Files/Services/Models/Recipe.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  Recipt.swift
+//  NetguruCollegeApp
+//
+
+import Foundation
+
+struct Recipe {
+    let title: String
+    let imageURL: String
+    let ingredients: String
+    let recipeURL: String
+}
+
+extension Recipe: Codable {
+    enum CodingKeys: String, CodingKey {
+        case title = "title"
```

Comment:
This `rawValue` is actually not needed here, as enum will manage it by itself when the value is the same as a case name.